templates/new_request_page.hbs

```diff
@@ -12,7 +12,7 @@
   </div>
 
   <h1>
-    {{t 'submit_a_request'}}
+    <div>Submit a Request or Provide Feedback</div>
```

Comment:
This as a different font size/styling than live theme at https://xpro.zendesk.com/hc/en-us/requests/new

**Before**
<img width="728" alt="image" src="https://github.com/mitodl/mitxpro-zendesk-theme/assets/34372316/7cd0555f-9ba1-4fca-91a1-0b59c8e5621e">


**Now:**
<img width="777" alt="image" src="https://github.com/mitodl/mitxpro-zendesk-theme/assets/34372316/9371be50-a503-44d8-a6de-38cba00ad934">

Reply:
Previously, the font-weight was set to 400 by the theme, which we did not customize. Now, the upstream master has changed the default font-weight to bold. We can choose to either adopt their new font-weight or retain the old font-weight.

Reply:
In this case, I think we should get a go ahead from @pdpinch.